id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: news_post
+title: "2020 Fukuoka Ruby Award Competition - Peserta akan dinilai oleh Matz"
+author: "Fukuoka Ruby"
+translator: "meisyal"
+date: 2019-10-16 00:00:00 +0000
+lang: id
+---
+
+Penggemar Ruby terhormat,
+
+Pemerintah Fukuoka, Jepang, bersama dengan "Matz" Matsumoto ingin mengundang
+Anda untuk mengikuti kompetisi Ruby berikut. Jika Anda pernah mengembangkan
+sebuah program Ruby yang menarik, dianjurkan untuk mengikuti kompetisi ini.
+
+2020 Fukuoka Ruby Award Competition - Hadiah Utama - 1 Juta Yen!
+
+Batas akhir pendaftaran: 11 Desember 2019
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz dan sebuah grup panelis akan memilih pemenang kompetisi ini. Haidah utama
```

Comment:
typo, "Hadiah" not "Haidah"

Reply:
Thank you for catching that. Fixed @kuntoaji